src/server_manager/web_app/app.ts

```diff
@@ -43,6 +43,7 @@ const CHANGE_HOSTNAME_VERSION = '1.2.0';
 // Date by which the data limits feature experiment will be permanently added or removed.
 const DATA_LIMITS_AVAILABILITY_DATE = new Date('2020-06-02');
 const MAX_ACCESS_KEY_DATA_LIMIT_BYTES = 50 * (10 ** 9);  // 50GB
+const HATS_DIALOG_DELAY_MS = 3000;
```

Comment:
Please add a comment

Reply:
This should actually be called PROMPT_IMPRESSION_DELAY, to follow Hats' terminology.

Reply:
Done.